lib/index.js

```diff
@@ -7,8 +7,7 @@
  * @typedef {BufferEncoding|{encoding?: null|BufferEncoding, flag?: string}} ReadOptions
  * @typedef {BufferEncoding|{encoding?: null|BufferEncoding, mode: Mode?, flag?: string}} WriteOptions
  *
- * @typedef {string|Uint8Array} Path Path of the file.
- * @typedef {Path|URL|Options|VFile} Compatible Things that can be
+ * @typedef {Value|URL|Options|VFile} Compatible Things that can be
```

Comment:
> However, only a string is accepted as a path. A buffer is allowed too.

*Both* are accepted as a path.

---

Hmm. While the expanded types are the same (string|buffer or string|uint8) for `Value` and `Path`, it becomes confusing when they’re both called the same here, as this package’s function is exactly not *not* accept a value, but instead accept a path.

Is it clearer to still have a thing called `Path` and a note?

```suggestion
 * @typedef {URL|Value} Path Path of the file.
 *   Note: `Value` is used here because it’s a smarter `Buffer`
 * @typedef {Path|Options|VFile} Compatible Things that can be
 ```